lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,31 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  end
+
+  def self.by_username(username)
+    user = User.find_by(username: username)
+    { user: user.public_user_attributes,
+      statistics: user_problem_statistcs(user) }
+  end
+
+  def self.problem_total(track)
+    track.problems.count
```

Comment:
Is this used in more than one place? I don't think the name is more expressive than the implementation, so I'd probably use `track.problems.count` inline where it's used.